### PR TITLE
Bugfixing/duplicating features

### PR DESCRIPTION
**Pull request by inacht**
Fix some bugs that occured when duplicating features.

Important hint: Here are some changes that cause a bc break when using a configured uri for downloading files. The uri for downloading files should be configered now for the schema featureType like this:
downloadFileUri: 'uri'

### CODE REVIEW

**Comment by LazerTiberius** [comment]
Can you explain the bugs a little bit? Does this fix that the Id will also be duplicated?

**Comment by inacht** [comment]
@LazerTiberius The duplicated id/entry was not considered as a bug (this was said so by the project management).

The following issues are fixed with this:
* configured  copy data field values will be used now (so they overwrite the corresponing form field values of the feature)
* configured copy rules will be applied
* missing semicolons added
* ensure feature.oldGeom has a non-empty value, so that no errors occur on duplicating

**Comment by frwg** [comment]
@inacht 
I don't know the exact requirements, but I would usually assume that it's the copied feature's values that overwrite the default values of a copied object.
As to feature.oldGeom - it was probably meant to keep the old position of a feature that is to be dragged, but it doesn't work (at least currently) since it is initiallized with .x and .y properties of an OpenLayers.Geometry Object - which simply do not exist.

**Comment by LazerTiberius** [comment]
@frwg It does work for points only. There should be a check for the feature geometry type indeed.

File: Resources/public/mapbender.element.digitizer.js [hunk 1]
```diff
@@ -3306,17 +3308,17 @@
             if(eventData.origin === 'cancel-button'){
                 this.currentPopup.popupDialog('close');
             }
-            if(!feature.hasOwnProperty('isNew') && feature.hasOwnProperty("oldGeom")){
+            if(!feature.hasOwnProperty('isNew') && feature.hasOwnProperty('oldGeom') && feature.oldGeom){
```
Comment (LazerTiberius) [inline Resources/public/mapbender.element.digitizer.js]: You can safely remove the hasOwnProperty check. It is redundant.

**Comment by inacht** [comment]
@LazerTiberius  unnecessary hasOwnProperty check was removed

**Pull request closed without merging**